docs/create_topology.md

```diff
@@ -117,9 +117,16 @@ Field           | Type       | Description
 
 Field  | Type      | Description
 ------ | --------- | ----------------------------------------------------
-`kind` | string    | Name of the controller type. The current options currently are `IxiaTG`, `SRLinux`, `CEOSLab`, and `Lemming`.
+`kind` | string    | Name of the controller type. The current options currently are `Cdnos`, `IxiaTG`, `SRLinux`, `CEOSLab`, and `Lemming`.
 `spec` | yaml.Node | Fields that set the options for the controller type.
 
+##### Cdnos
+
+Field           | Type       | Description
+--------------- | ---------- | -----------
+`operator`      | string     | Path of the yaml file to create a Cdnos operator in the cluster. The validated operator for use with KNE can be found [here](https://github.com/openconfig/kne/tree/main/manifests/controllers/cdnos/manifest.yaml).
+~~`manifests`~~ | ~~string~~ | ~~Path of the directory holding the manifests to create a CEOSLab operator in the cluster. The directory is expected to contain a file with the name `manifest.yaml`.~~
```

Comment:
this can be removed after the field is removed

Reply:
removed